src/main/scala/cromwell/server/WorkflowManagerSystem.scala

```diff
@@ -12,15 +14,15 @@ trait WorkflowManagerSystem {
   implicit val actorSystem = ActorSystem(systemName)
 
   // For now there's only one WorkflowManagerActor so no need to dynamically name it
-  lazy val workflowManagerActor = actorSystem.actorOf(WorkflowManagerActor.props(dataAccess, backend), "WorkflowManagerActor")
+  lazy val workflowManagerActor = actorSystem.actorOf(WorkflowManagerActor.props(backend), "WorkflowManagerActor")
 
   // Lazily created as the primary consumer is the workflowManagerActor.
-  private lazy val dataAccess: DataAccess = DataAccess()
+  private lazy val dataAccess: DataAccess = DataAccess.instance
 
   /**
    * Should be called after the system is no longer in use.
    *
    * @return Non-blocking future that will eventually shut down this instance or return an error.
    */
-  def shutdown() = dataAccess.shutdown()
+  def shutdown() = Future.successful(()) //dataAccess.shutdown()
```

Comment:
We probably should. There's a ticket deep in the backlog to have some sort of proper shutdown functionality that latches in on the process' shutdown hook but it hasn't been a priority